packages/scxa-marker-gene-heatmap/src/clustersView/ClustersHeatmapView.js

```diff
@@ -77,9 +75,9 @@ class HeatmapView extends React.Component {
   }
 
   render() {
-    const { data, filteredData, selectedClusterId, isLoading, hasError } = this.state
+    const { data, filteredData, isLoading, hasError } = this.state
     const { wrapperClassName, plotWrapperClassName } = this.props
-    const { ks, ksWithMarkers, selectedK, onSelectK } = this.props
+    const { ks, ksWithMarkers, selectedColourBy, onChangeMarkerGeneFor, onChangeColourBy, metadata, cellTypes, selectedColourByCategory, selectedClusterId} = this.props
```

Comment:
The variable names `selectedColourBy`, `onChangeColourBy` and `selectedColourByCategory` were appropriate in the context of the cell plots because the scatter plot series would get different colours depending on these values. However, in the heatmap it’s not the case. Could you rename these variables to something better?